src/Cryptogram.java

```diff
@@ -0,0 +1,36 @@
+import java.io.*;
+import java.util.*;
+
+public class Cryptogram{
+    
+    public List<String> cryptogram;
+    public HashMap<Character, Integer> frequencyCount;
+
+    public Cryptogram(String cryptogramFile) throws FileNotFoundException{
+        String cryptogramString = FileReader.readFile(cryptogramFile);
```

Comment:
The cryptogram shouldn't take in a path as the argument, but instead the string returned by `readFile`.